lib/kafka_rest_client.js

```diff
@@ -151,6 +154,9 @@ KafkaRestClient.prototype.produce = function produce(produceMessage, callback) {
         });
     } else {
         self.produceWithRetry(produceMessage, 0, callback);
+        if (this.statsd) {
+          this.statsd.increment(self.metricsPrefix + '.produced'+ produceMessage.topic);
```

Comment:
In the naming convention I used for python event (produced/retry/latency)is in the end